src/models/Offer.js

```diff
@@ -62,7 +62,7 @@ const OfferSchema = new Schema({
     },
 
     isPaid: { type: Boolean },
-    vacancies: { type: Number },
+    vacancies: { type: Number, min: OfferConstants.vacancies.min },
```

Comment:
Does this allow an empty value as well?

Reply:
I took this form the docs:
`Validators are not run on undefined values. The only exception is the [required validator](https://mongoosejs.com/docs/api.html#schematype_SchemaType-required).` 

[here](https://mongoosejs.com/docs/schematypes.html#number-validators)